tools/csv-merger/csv-merger.go

```diff
@@ -855,6 +862,8 @@ func main() {
 		log.Fatal("--noobaa-db-image is required")
 	} else if *ocsContainerImage == "" {
 		log.Fatal("--ocs-image is required")
+	} else if *mustGatherImage == "" {
+		log.Fatal("--must-gather-image is required")
```

Comment:
Why? We don't need must-gather image to generate CSV whatsoever.
This should be kept optional and only be used by whoever needs it.